fw/http_msg.c

```diff
@@ -815,6 +820,47 @@ __hdr_sub(TfwHttpMsg *hm, const TfwStr *hdr, unsigned int hid)
 	return TFW_PASS;
 }
 
+/**
+ * Substitute request method
+ *
+ * For a while it's just a custom realization for substituting dest method with sub method
+ * (Implemented to substitute PURGE with GET in X-Tempesta-Cache processing)
+ */
+int
+tfw_http_req_sub(TfwHttpMsg *hm, unsigned char dest_meth, unsigned char sub_meth)
```

Comment:
We use the `tfw_http_req` prefix around the code to reference HTTP requests logic, so it makes sense to add some more sense to the name, e.g. `tfw_http_req_meth_sub()`.